content/Developement/gettingStarted.md

```diff
@@ -2,6 +2,7 @@
 title: "GettingStarted"
 date: 2023-01-10T12:35:11+05:30
 draft: false
+aliases: [/GettingStarted]
```

Comment:
Was there ever a link:
`https://hive.apache.org/GettingStarted`?

I just tried the following and does not work either.
`https://apache.github.io/hive-site/GettingStarted`

Reply:
I think we don't need an alias. We just need to fix:
https://github.com/apache/hive-site/blob/6097c3f2cf8e0186a39f172ac8165509b1457a6e/content/Developement/gettingStarted.md?plain=1#L55

to point to the wiki:
https://cwiki.apache.org/confluence/display/Hive/GettingStarted

Reply:
You are right, aliases is not needed for the Getting started page. We need these for the other pages at least until google claws the hive website and updates its search index.

Pointing the wiki to https://cwiki.apache.org/confluence/display/Hive/GettingStarted  is being addressed here by Mahesh - https://issues.apache.org/jira/browse/HIVE-26983

Thanks!